### PR TITLE
feat(@wallet): multi network history

### LINKED ISSUE #4749: add multinetwork functionality to activities
### Description

with a flag to enable the multinetwork functionality, we should be able to select one or multiple networks and see the agregate activities

### PR DESCRIPTION
**Pull request by alaibe**
Multinetwork:
* for estimate time
* transaction history (NO UI but data is loaded and displayed)
* for browser
* for node configuration

Remove toggle for multi network
Clean all previous network code

status-go PR: https://github.com/status-im/status-go/pull/2733

fixes #6271
fixes #4749

### CODE REVIEW

**Review by jrainville — APPROVED** [review]
Good work!

File: ui/app/AppLayouts/Browser/popups/BrowserWalletMenu.qml [hunk 1]
```diff
@@ -70,10 +70,10 @@ Popup {
         StatusBaseText {
             id: networkText
             text: {
-                switch (Web3ProviderStore.currentNetwork) {
-                case Constants.networkMainnet: return qsTr("Mainnet");
-                case Constants.networkRopsten: return qsTr("Ropsten");
-                default: return qsTrId("Unknown")
+                switch (Web3ProviderStore.chainName) {
```
Comment (Khushboo-dev-cpp) [inline ui/app/AppLayouts/Browser/popups/BrowserWalletMenu.qml:73]: why does this show only these two networks statically? Mainnet and Ropsten? what about the others?

**Review by Khushboo-dev-cpp — APPROVED** [review]
Awesome work! You made it look so easy!

File: ui/app/AppLayouts/Browser/popups/BrowserWalletMenu.qml (hunk 1 above)
Comment (alaibe) [reply ui/app/AppLayouts/Browser/popups/BrowserWalletMenu.qml:73]: on browser we decided to have only those 2, similar to how it works with ENS/stickers

Comment (Khushboo-dev-cpp) [reply ui/app/AppLayouts/Browser/popups/BrowserWalletMenu.qml:73]: I didnt know that !! :D

**Comment by anastasiyaig** [comment]
dont merge it yet , lets do after release

**Review by anastasiyaig — APPROVED** [review]

**Pull request merged**